Makefile

```diff
@@ -123,34 +126,47 @@ rocm-gfx900: $(ROCM_DEPS)  ## Build container for AMD ROCm gfx900 (untested)
 	$(call build-rocm,gfx900,9.0.0)
 
 .PHONY: rocm-toolbox
-toolbox-rocm:  ## Create AMD ROCm toolbox from container
+toolbox-rocm: check-toolbox ## Create AMD ROCm toolbox from container
 	toolbox create --image $(CONTAINER_PREFIX):rocm $(TOOLBOX)
 
 .PHONY: toolbox-rm
-toolbox-rm:  ## Stop and remove toolbox container
+toolbox-rm: check-toolbox ## Stop and remove toolbox container
 	toolbox rm -f $(TOOLBOX)
 
 ##@ Development
 
 .PHONY: tests
-tests: ## Run tox -e unit against code
+tests: check-tox ## Run tox -e unit against code
 	tox -e py3-unit
 
 .PHONY: verify
-verify: ## Run tox -e fmt,lint,spellcheck against code
+verify: check-tox ## Run tox -e fmt,lint,spellcheck against code
 	tox p -e ruff,fastlint,spellcheck
 
 .PHONY: spellcheck-sort
 spellcheck-sort: .spellcheck-en-custom.txt
 	sort -d -o $< $<
 
 .PHONY: man
-man:
+man: check-tox
 	tox -e docs
 
 .PHONY: docs
 docs: man ## Run tox -e docs against code
 
+##@ Validation
+check-tox:
```

Comment:
The check targets should be marked as phony targets. They do not create files with the target name.